src/Projections/Test/TestHost.ProbeByHost.cs

```diff
@@ -21,8 +21,10 @@ internal interface IActivationFactory
 }
 
 namespace ABI.Windows.Foundation
-{
-    [global::WinRT.ObjectReferenceWrapper(nameof(_obj))]
+{
+#if !NET6_0
```

Comment:
Why not !NET here for consistency with the others.

Reply:
I didn't realize `NET` was also defined here, fixed! Thank you 🙂